packages/cli/create-strapi-app/templates/example/package.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
This app is supposed to be written in TS, but files in the src/api folder are .js